workflow/rocoto/gfs_tasks.py

```diff
@@ -2180,8 +2163,12 @@ def arch(self):
                 dep_dict = {'type': 'task', 'name': f'{self.cdump}wavepostbndpnt'}
                 deps.append(rocoto.add_dependency(dep_dict))
         if self.app_config.do_ocean:
-            if self.app_config.mode in ['forecast-only']:  # TODO: fix ocnpost to run in cycled mode
-                dep_dict = {'type': 'metatask', 'name': f'{self.cdump}ocnpost'}
+            if self.app_config.mode in ['forecast-only']:
```

Comment:
Why are these only dependencies for forecast-only?

Reply:
on the LHS, this was in `forecast-only`, so I retained it.
In the cycled mode, I don't think we are/were running `ocnpost`.  We should clarify this regardless.
Are there ocean/ice products for `RUN=gdas`? @GwenChen-NOAA

Reply:
@aerorahul, for GDAS, [the current plan](https://docs.google.com/document/d/1AwYbHArLJ6KvD0HW5bUbNOiFJaVAWgWEmnMQ2g076TA/edit, Pages 6-7) is to generate background (e.g., gdas.tHHz.ocnf006.nc) and analysis (e.g., gdas.tHHz.ocnanl.nc) files only, and they will not be distributed through NOMADS. The background and analysis files will be outputted in model resolution, so there is no plan to run `ocnpost` for GDAS ocean/ice at this moment.